### ISSUE TITLE
customTitle doesn't show

### ISSUE THREAD

**Issue by dimoreira** [Kureev/react-native-navbar#10]
I'm using `react-native-navbar` to load an app logo to the navbar.
The **customTitle** attribute doesn't work either to show an `<View />` component on the navbar.

Using this demo:

```javascript
/* index.ios.js */
...

<Navigator
    renderScene={this._renderScene}
    initialRoute={{
        component: FacebookConnectPage,
        navigationBar:
            <NavigationBar
                customTitle={LogoNavTitle}
                backgroundColor="#00605e"
            />
    }}
/>
```

```javascript
/* LogoNavTitle.js */
...

<View style={{ flexDirection: 'row', height: 60, backgroundColor: '#fff' }}>
</View>
```

I'm trying to just show an view with white background to check if it works and that don't render either.

**Comment by Kureev** [comment]
You need to pass a link to the instance of component. I made a [tiny example of index.ios.js](https://gist.github.com/Kureev/929f351064b63f8170e0) to show how it should looks like. Let me know if you'll have any issues with it.

**Comment by dimoreira** [comment]
@Kureev Great man! That worked!

One more doubt:

If I want to make the customTitle customizable for each new view in the route stack, can I pass a title as navigationBar prop? What is the best approach for this?

**Comment by Kureev** [comment]
If you want to pass a title, you can use `title` prop. You can check the full list here: https://github.com/Kureev/react-native-navbar#component-props

**Comment by dimoreira** [comment]
@Kureev So, everytime I pass the `customTitle` along `title`, the `customTitle` prop will take precedence?

**Comment by Kureev** [comment]
Yes, and it works the same way for all custom components.

**Comment by Kureev** [comment]
I've just released 0.4.0. You should check the `examples` folder in it. Let me know, if you'll still have questions.
